src/mqtt_client/AtlasMqttException.h

```diff
@@ -0,0 +1,17 @@
+#ifndef __ATLAS_MQTT_EXCEPTION_H__
```

Comment:
Please remove this logger and use the existing AtlasLogger

Reply:
Code updated with the integrated logger.